Modules/Invoice/Resources/views/subviews/edit/invoice-details.blade.php

```diff
@@ -138,6 +138,13 @@
                     </div>
                 </div>
 
+                <div class="form-group" v-if="this.client.type != 'indian'">
+                    <div class="d-flex">
+                        <label for="conversion_rate" class="mr-4 pt-1 field-required">Conversion Rate:</label>
+                        <input type="text" class = "form-control w-272 ml-auto" name="conversion_rate" v-model = 'conversion_rate'required="required">
+                    </div>
```

Comment:
Is d-flex required? What will happen if I only do this?
```suggestion

                    <label for="conversion_rate" class="mr-4 pt-1 field-required">Conversion Rate:</label>
                    <input type="text" class="form-control w-272 ml-auto" name="conversion_rate" v-model="conversion_rate" required="required">

```

Reply:
I see `<label for="conversion_rate"` but couldn't find `<input id="conversion_rate"` in the `input` tag. What is the purpose of `for` attribute in `label` tag?

Reply:
I used d-flex and ``` for ``` attribute in ```label``` tag because they are used in all the other fields in the form and I tried to follow the same structure.

Reply:
Your answer is not relevant to the question @kuldeep3 asked. You can google search the purpose of `for` attribute and its use and then you can understand better what Kuldeep is asking.

Reply:
> Is d-flex required? What will happen if I only do this?

@kuldeep3 Removing `d-flex` disturbs the UI as it the input field comes below the label.

Reply:
> I see `<label for="conversion_rate"` but couldn't find `<input id="conversion_rate"` in the `input` tag. What is the purpose of `for` attribute in `label` tag?

Resolved in this commit https://github.com/ColoredCow/portal/pull/856/commits/57553d9d9084cb90827045c707c18cde5ab17815